docs/samples/specification/azure_key_credential/generated/azure/key/credential/sample/aio/__init__.py

```diff
@@ -7,9 +7,16 @@
 # --------------------------------------------------------------------------
 
 from ._auto_rest_head_test_service import AutoRestHeadTestService
+
+try:
+    from ._patch import __all__ as _patch_all
+    from ._patch import *  # pylint: disable=unused-wildcard-import
+except ImportError:
+    _patch_all = []
+from ._patch import patch_sdk as _patch_sdk
 __all__ = ['AutoRestHeadTestService']
+__all__.extend(_patch_all)
 
 # `._patch.py` is used for handwritten extensions to the generated code
 # Example: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/customize_code/how-to-patch-sdk-code.md
-from ._patch import patch_sdk
-patch_sdk()
+_patch_sdk()
```

Comment:
After this PR, there will be two ways to override generated code:
(1) define `__all__` in `_patch.py`
(2) use importlib to override in `patch_sdk`
The latter will override the former so we need to emphasize it in document.

Reply:
@msyyc the hope is really to not need patch_sdk anymore. It's a last resort (meaning only if you really can't do it in nice customizations way, we'll use it). 

I've included your old doc at the end of the customizations file: https://github.com/Azure/autorest.python/pull/1123/files#diff-386e77f016c4001b9af07398750a1ecb080a5235ea22eb027bc16a2e37022ea2R235

I may need to update though, since a couple of the examples here are showing examples where we should really use customizations, so will do a PR for that soon

Reply:
updated, you can look at any `def patch_sdk` in the patch files. Let me know if you like it!